app/root.tsx

```diff
@@ -1,178 +1,42 @@
-import {
```

Comment:
I don't agree with most of the changes in this file. I still think we need `links`, `App`, `ErrorBoundary` & `CatchBoundary`.